RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -73,16 +73,17 @@ def _updateOutput( era, outputPSets, commands):
         'keep *_hgcalLayerClusters_*_*',
         'keep *_hgcalMultiClusters_*_*',
         'keep *_iterHGCalMultiClusters_*_*',
-
+        'drop DetIdfloatstdmap_hgcalLayerClusters_*_*',
     ]
 )
 phase2_hgcal.toModify( RecoLocalCaloRECO,
     outputCommands = RecoLocalCaloRECO.outputCommands + ['keep *_HGCalRecHit_*_*',
                                                          'keep *_hgcalLayerClusters_*_*',
+                                                         'drop DetIdfloatstdmap_hgcalLayerClusters_*_*',
                                                          'keep *_hgcalMultiClusters_*_*',
                                                          'keep *_iterHGCalMultiClusters_*_*'] )
 # don't modify AOD for HGCal yet, need "reduced" rechits collection first (i.e. requires reconstruction)
-phase2_hgcal.toModify( RecoLocalCaloAOD, outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_HGCalRecHit_*_*','keep *_hgcalLayerClusters_*_*'] )
+phase2_hgcal.toModify( RecoLocalCaloAOD, outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_HGCalRecHit_*_*','keep *_hgcalLayerClusters_*_*','drop DetIdfloatstdmap_hgcalLayerClusters_*_*'])
```

Comment:
please add some linebreaks here so that this long line fits on the screen (as multiple lines)

Reply:
@rovere 
ping
in case this got lost